modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -82,6 +82,22 @@ def get_appointments(start_date, end_date, statuses = nil, pagination_params = {
         }
       end
 
+      def get_all_appointments
+        start_date = (Time.zone.today - 100.years).in_time_zone
+        end_date = (Time.zone.today + 100.years).in_time_zone
+        params = date_params(start_date, end_date)
+
+        with_monitoring do
+          response = if Flipper.enabled?(APPOINTMENTS_USE_VPG, user)
```

Comment:
I tried to find if these endpoints support pagination and found conflicting information. Do you know if these will return pagination which will mean you will not get all appointments?

Reply:
@carlosfelix2 I'm not really sure tbh, the call to `get_appointments` in the index action allows for the passing of pagination params in the request but does not require them. I think we could safely follow this pattern and have our bases covered.